libs/src/services/schemaValidator/schemas/userSchema.json

```diff
@@ -1,235 +1,232 @@
+          "type": ["string", "null"],
+          "default": null,
+          "$ref": "#/definitions/CID"
+        },
+        "cover_photo": {
+          "type": ["string", "null"],
+          "default": null,
+          "$ref": "#/definitions/CID"
+        },
+        "cover_photo_sizes": {
+          "type": ["string", "null"],
+          "default": null,
+          "$ref": "#/definitions/CID"
+        },
+        "bio": {
+          "type": ["string", "null"],
+          "default": null
+        },
+        "location": {
+          "type": ["string", "null"],
+          "default": null
+        },
+        "associated_wallets": {
+          "$ref": "#/definitions/AssociatedWallets"
         },
-        "PlaylistIdentifier": {
-            "properties": {
-                "type": {
-                    "type": "string",
-                    "const": "playlist"
-                },
-                "playlist_id": {
-                    "type": "integer"
-                }
-            }
-
-        },
-        "ExplorePlaylistIdentifier": {
-            "properties": {
-                "type": {
-                    "type": "string",
-                    "const": "explore_playlist"
-                },
-                "playlist_id": {
-                    "type": "string",
-                    "pattern": "^[a-zA-Z \/-?]+$"
-                }
-            }
-        },
-        "PlaylistLibraryIdentifier": {
+        "associated_sol_wallets": {
+          "$ref": "#/definitions/AssociatedSOLWallets"
+        },
+        "collectibles": {
+          "$ref": "#/definitions/Collectibles"
+        },
+        "playlist_library": {
+          "$ref": "#/definitions/PlaylistLibrary"
+        },
+        "events": {
+          "$ref": "#/definitions/Events"
+        },
+        "is_deactivated": {
+          "type": "boolean",
+          "default": false
+        },
+        "artist_pick_track_id": {
+          "type": ["integer", "null"],
+          "default": null
+        }
+      },
+      "required": [
+        "bio",
+        "cover_photo",
+        "cover_photo_sizes",
+        "location",
+        "name",
+        "profile_picture",
+        "profile_picture_sizes",
+        "artist_pick_track_id"
+      ],
+      "title": "User"
+    },
+    "CID": {
+      "type": ["string", "null"],
+      "minLength": 46,
+      "maxLength": 46,
+      "pattern": "^Qm[a-zA-Z0-9]{44}$",
+      "title": "CID"
+    },
+    "AssociatedWallets": {
+      "type": ["object", "null"],
+      "default": null,
+      "patternProperties": {
+        "^0x[a-fA-F0-9]{40}$": { "$ref": "#/definitions/WalletMetadata" }
+      },
+      "additionalProperties": false,
+      "properties": {},
+      "required": [],
+      "$comment": "",
+      "title": "AssociatedWallets"
+    },
+    "AssociatedSOLWallets": {
+      "type": ["object", "null"],
+      "default": null,
+      "patternProperties": {
+        "^[a-zA-Z0-9]{43,44}$": { "$ref": "#/definitions/WalletMetadata" }
```

Comment:
This should be 32 to 44 characters
ref docs: https://docs.solana.com/integrations/exchange#basic-verification

Reply:
I believe it is base58 encoded right? so maybe match it exactly to their regex 
`[1-9A-HJ-NP-Za-km-z]{32,44}`

Reply:
ah perfect, thanks for looking into that!

Reply:
@jowlee updated